advanced_calculator.py

```diff
@@ -0,0 +1,391 @@
+# This is like making a package.lock file for npm package.
+# Yes, I should be making it.
+__author__ = "Nitkarsh Chourasia"
+__version__ = "0.0.0"  # SemVer # Understand more about it
+__license__ = "MIT"  # Understand more about it
+# Want to make it open source but how to do it?
+# Program to make a simple calculator
+# Will have to extensively work on Jarvis and local_document and MongoDb and Redis and JavaScript and CSS and DOM manipulation to understand it.
+# Will have to study maths to understand it more better.
+# How can I market gtts? Like showing used google's api? This is how can I market it?
+# Project description? What will be the project description?
+
+from numbers import Number
+from sys import exit
+import colorama as color
```

Comment:
Also adding terminal based controls

Reply:
Just to exit the program out of while loop which takes continuous inputs like a actual calculator and when power is pressed -> sys.exit() is executed, suppose.